prow/plugins/wip/wip-label.go

```diff
@@ -37,7 +37,7 @@ const (
 )
 
 var (
-	titleRegex = regexp.MustCompile(`(?i)^(\[WIP\]|WIP)`)
+	titleRegex = regexp.MustCompile(`(?i)^\W?WIP\W`)
```

Comment:
We will stop matching bare prefixes if the second regexp is removed. I propose adding a space in the end and making the first \W optional.  `(?i)^(\W?WIP\W|WIP )`

It also matches:
```
wip blabla
wip: blabla
```

Reply:
Ah you already add the first as optional, nvmd